### DISCUSSION TITLE
Monogame Plugin

### DISCUSSION THREAD

**Discussion by GorEldeen** [consulo/consulo#523] [Ideas]
Can you add monogame support for Consulo? I've been trying every c# ide available for linux, But only this ide stood out with perfection, But i can't find support for monogame, Also don't know how to use it with it, Can someone help me?

**Comment by VISTALL** [comment]
Hello. I'm open for suggestion's what you need for it. Monogame share same project system like other C# plugins. If you found bugs - you can write about them.

Since I'm not user of Monogame - I can't say what you need.

I saw only one plugin for Monogame (https://github.com/seclerp/rider-monogame) - which add some known extensions, and some file templates, that's all.
